install.sh

```diff
@@ -58,8 +58,25 @@ echo " Hostname: $HOSTNAME_FQDN"
 echo $LINE
 
 # Detect playbook path option
-if [ $1 ]; then
-    PLAYBOOK_PATH=$1
+while [ $# -gt 0 ]; do
+  case "$1" in
+    --playbook=*)
+      PLAYBOOK_PATH="${1#*=}"
+      ;;
+    --server-webserver=*)
+      SERVER_WEBSERVER="${1#*=}"
+      echo " Installing webserver $SERVER_WEBSERVER";
+      ;;
+    *)
+      printf "***************************\n"
+      printf "* Error: Invalid argument.*\n"
+      printf "***************************\n"
```

Comment:
Let's keep consistent with the rest of the script, and be verbose to let the user know what they did wrong:

```
echo $LINE
echo " Error: You must specify 'apache' or 'nginx' as the server-webserver option."
echo $LINE
```